query.sql

```diff
@@ -0,0 +1,531 @@
+        | 100061   | 02                | 02                | DEMO_UNIT_5 | Demo Product Unit Five  | 1.000000 |    55.09000 | ItemProduct       | null               |
+        | 100064   | 01                | 01                | DEMO_UNIT_5 | Demo Product Unit Five  | 3.000000 |    12.07000 | ItemProduct       | null               |
+        | 100064   | 02                | 02                | DEMO_UNIT_2 | Demo Product Unit Two   | 2.000000 |     6.33000 | ItemProduct       | null               |
+        | 100102   | 01                | 01                | DEMO_UNIT_1 | Demo Product Unit One   | 3.000000 |    16.99000 | ItemProduct       | null               |
+        | 100102   | 02                | 01                | DEMO_UNIT_2 | Demo Product Unit Two   | 3.000000 |    18.99000 | ItemProduct       | null               |
+        | 100102   | 03                | 01                | null        | Sales Tax               | 3.000000 |     2.33000 | ItemSalesTax      | 02                 |
+        | 100102   | 04                | 02                | DEMO_UNIT_3 | Demo Product Unit Three | 1.000000 |    14.99000 | ItemProduct       | null               |
+        | 100153   | 01                | 01                | DEMO_UNIT_3 | Demo Product Unit Three | 1.000000 |    14.99000 | ItemProduct       | null               |
+        | 100153   | 02                | 01                | null        | Sales Tax               | 1.000000 |     1.33000 | ItemSalesTax      | 01                 |
+        | 100153   | 03                | 01                | DEMO_UNIT_4 | Demo Product Unit Four  | 1.000000 |    12.99000 | ItemProduct       | null               |
+        | 100153   | 04                | 01                | null        | Sales Tax               | 1.000000 |     2.33000 | ItemSalesTax      | 03                 |
+        | 100255   | 01                | 01                | DEMO_UNIT_4 | Demo Product Unit Four  | 3.000000 |    12.99000 | ItemProduct       | null               |
+        | 100255   | 02                | 01                | null        | Sales Tax               | 3.000000 |     1.33000 | ItemSalesTax      | 01                 |
+        | 100255   | 03                | 02                | DEMO_UNIT_5 | Demo Product Unit Five  | 3.000000 |    10.99000 | ItemProduct       | null               |
+        +----------+-------------------+-------------------+-------------+-------------------------+----------+-------------+-------------------+--------------------+
+
+
+2. List the person details for all females.
+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+Query: SELECT FIRST_NAME,MIDDLE_NAME,LAST_NAME,BIRTH_DATE,
+       EMPLOYMENT_STATUS_ENUM_ID,OCCUPATION
+       FROM person
+       WHERE SALUTATION = 'Ms';
```

Comment:
You should not use SALUTATION to filter the record instead of this you should check the GENDER.